src/bauhaus/bauhaus.c

```diff
@@ -1684,11 +1690,14 @@ static gboolean dt_bauhaus_popup_draw(GtkWidget *widget, cairo_t *crf, gpointer
           if(entry->alignment == DT_BAUHAUS_COMBOBOX_ALIGN_LEFT)
             label_width = show_pango_text(w, context, cr, entry->label, 0, ht * k, max_width, FALSE);
```

Comment:
0 should probably be INNER_PADDING to avoid the left label to be close to the border of left windows.

Reply:
If revised my pull request and fixed various padding issues for bauhaus combobox. Looks much better now.

Reply:
@rabauke : The width of the combo is still very big, can this be fixed somehow? Or maybe you forgot part of your patch ?

Reply:
Well not sure now, this kind of combobox are used in many placed in dt, so you current proposal is what seems correct. I'll review again.

Reply:
Actually, I have not changed anything in particular to adjust the total width of the combobox. The width of the combobox automatically adjusts to the width of the (right) panel. It has the same width as a slider. This behavior is observed with and without my patch and applies to all comboboxes in darktable. I you feel the combobox is too broad, you may narrow the size of your right panel.